lib/otto_api/client.ex

```diff
@@ -47,9 +44,15 @@ defmodule OttoApi.Client do
   defp request_with_body(verb, client, path, body) do
     with {:ok, json} <- Jason.encode(body) do
       {:ok, response} =
-        apply(@http_client, verb, [full_url(client.base_url, path), json, headers(client)])
+        apply(@http_client, verb, [full_url(client.base_url, path), json, headers(client), []])
+
+      body =
+        case response.body do
+          "" -> %{}
+          _ -> Jason.decode(response.body)
```

Comment:
I think `Jason.decode/1` returns a tuple. I think this would result in the response potentially being a nested `:ok` tuple like this:
```
{:ok, %{}} || {:ok, {:ok, dict()}}
```

If we want to crash on JSON errors, should we use `Jason.decode!/1` instead?

Reply:
Hmm, I think you're right.  Now I have to figure out why my tests are passing anyway.